caput/scripts/runner.py

```diff
@@ -34,7 +29,6 @@ def cli():
 )
 def lint_config(configfile):
```

Comment:
I was surprised to learn that `click.Path` arguments don't return a `pathlib.Path` object. You'd think they would, but they can't because they need to support Python 2 apparently.